### ISSUE TITLE
The server crashes

### ISSUE THREAD

**Issue by Panzimy** [ProgrammerLP/Create-Trainutilities#7]
Say what I need to do

**Comment by Panzimy** [comment]
And yeah the mod on server crashes

**Comment by ProgrammerLP** [comment]
Can you give me more information, like a crash report or server logs?
Otherwise i cant identify the problem

**Comment by Panzimy** [comment]
Ok, I will give it as soon as I can

**Comment by Panzimy** [comment]
[2024-11-04-3.log.gz](https://github.com/user-attachments/files/17622183/2024-11-04-3.log.gz)
Here is log where the error is presented

**Comment by Panzimy** [comment]
Yeah and that is all what I know. I started my server and after some time the server will not start.

**Comment by ProgrammerLP** [comment]
> And I can do a Czech translation

That would be nice, can you open a seperate issue for that?

**Comment by ProgrammerLP** [comment]
> Can confirm the server also crashing after updating (from 2.2.0 to 2.3.0) on Fabric, comparing both of the logs, not sure if it's the same issue though:
> 
> Server log:
> https://gist.github.com/GameBurrow/51574cad0215f80d779dff7fee4ae0de
> 
> Reverting only this mod to 2.2.0, fixes the issue

Thanks i will fix it soon

**Comment by Panzimy** [comment]
Yeah if I download 2.2.0 everything is OK

**Comment by ProgrammerLP** [comment]
problem fixed i will release a patch soon!
